src/main/resources/assets/lambdabettergrass/lang/es_mx.json

```diff
@@ -0,0 +1,12 @@
+{
+  "lambdabettergrass.menu.info.1": "El mod usa archivos de paquetes de recursos para determinar qué bloques se ven afectados.",
+  "lambdabettergrass.menu.info.2": "Esto significa habilitar o deshabilitar la función Better Grass(mejor pasto) en bloques",
+  "lambdabettergrass.menu.info.3": "debe modificarse con paquetes de recursos.",
+  "lambdabettergrass.menu.info.read_more": "Leer más en: %s",
+  "lambdabettergrass.menu.title": "LambdaBetterGrass - Configuraciones",
+  "lambdabettergrass.menu.title.info": "¿Cómo funciona el mod?",
+  "lambdabettergrass.option.better_snow": "Better Snow(mejor nieve)",
```

Comment:
I am a bit confused, is it normal that the thing between parenthesis is directly stuck next to the "Better snow" thing?

Reply:
I was considering if I should translate the actual feature name itself due to how English works. Considering the fact the feature name is capitalized. I can completely replace the feature name with an Spanish translation.

Reply:
Well you can translate it if you think it's more understandable, currently in Simplified Chinese and French translations those feature names are translated.